cmd/migratecheckpoint/main.go

```diff
@@ -110,7 +110,7 @@ func (m *Migrator) MigrateCustomPos(matches []string) {
 				if err != nil {
 					log.Printf("error creating a new DB client for host file checkpoints: %v", err)
 				}
-				err = client.Set("$.file_input.knownFiles", buf.Bytes())
+				err = client.Set("file_input.knownFiles", buf.Bytes())
```

Comment:
Have we _always_ had this issue? Is there a breaking change in stanza that occurred?

Reply:
Well, not sure tbh. Need to dig in through commit history.

Reply:
We don't have any tests for migration from SCK. I'll create a PR in the chart repo covering this part.
If I understand correctly, we're moving to Golang-based functional tests, right?

Reply:
@atoulme here's a PR to add functional test cases covering this. https://github.com/signalfx/splunk-otel-collector-chart/pull/1024
PTAL! Thanks